assets/css/main.css

```diff
@@ -302,32 +304,33 @@ h4 {
 }
 
 /* Cards */
-.card-group {
+/* .card-group {
   display: flex;
   flex-flow: row wrap;
   margin-top: 4rem;
   justify-content: space-between;
-}
+} */
 
 .card {
-  width: 45%;
-  display: flex;
+  width: 49%;
+  /* display: flex;
   flex-direction: column;
   min-width: 0;
   word-wrap: break-word;
   background-color: #fff;
   background-clip: border-box;
   border: 1px solid rgba(0,0,0,.125);
-  border-radius: .25rem;
-  margin-bottom: 2rem;
+  border-radius: .25rem; */
+  margin-top: 1rem;
+  margin-bottom: 1rem;
 }
```

Comment:
You could handle this with a class in the html `my-3`. The value for 3 - (by default) for classes that set the margin or padding to $spacer and the value for $spacer | 1rem (default), or any value > 0